src/test/java/teammates/ui/webapi/GetFeedbackResponsesActionTest.java

```diff
@@ -229,7 +229,7 @@ protected void testAccessControl_getNonExistingFeedbackResponse_shouldFail() {
                 Const.ParamsNames.FEEDBACK_QUESTION_ID, "randomNonExistId",
                 Const.ParamsNames.INTENT, Intent.INSTRUCTOR_SUBMISSION.toString(),
         };
-        verifyEntityNotFoundAcl(nonExistParams);
+        verifyHttpParameterFailureAcl(nonExistParams);
```

Comment:
Any idea why it gives a different error?

Reply:
For this test, it was previously for `entityNotFound` for a questionId. 

We call the old logic to try to fetch the question, which is null. 
Then, we try to call the new logic to fetch the question, but there will be a http param error instead as the old questionId is `String`, while the new questionId is `UUID`

Reply:
There should be no need for any changes to behaviour of existing API for GET actions.